p2p/stdnetwork/libp2p.go

```diff
@@ -0,0 +1,184 @@
+package stdnetwork
```

Comment:
I don't think stdnetwork is the right place here. It was meant for types that implement the `typesP2P.Network` interface (i.e. RainTreeNetwork, StdNetwork, etc...)

A separate packaged `p2p/libp2pnetwork` is probably best

Reply:
I moved it into `p2p/libp2p` and named the file `network.go`. Does that seem reasonable to you?

Reply:
Yes